app/jobs/submit_folio_request_job.rb

```diff
@@ -93,13 +93,16 @@ def place_title_hold
 
     def pickup_location_id
       @pickup_location_id ||= begin
-        code = Settings.libraries[request.destination].folio_pickup_service_point_code
-        code ||= Settings.libraries['GREEN'].folio_pickup_service_point_code
-
-        Folio::Types.instance.service_points.values.find { |v| v.code == code }&.id
+        code = service_point_code(request.destination)
+        Folio::Types.instance.service_point_id(code)
       end
     end
 
+    def service_point_code(destination)
+      # Check if comparable service point code exists, otherwise return default
+      Folio::Types.instance.valid_service_point_code?(destination) ? destination : Settings.folio.default_service_point
```

Comment:
At least for the transition period, I think we probably want to fall back on the `Settings.libraries` lookup (at least to handle the backlog of requests that come in after Symphony is done and FOLIO isn't ready)

```suggestion
      Folio::Types.instance.valid_service_point_code?(destination) ? destination :  Settings.libraries[destination]&.folio_pickup_service_point_code || Settings.folio.default_service_point
```

Reply:
Sounds good.